crowdstrike/client.go

```diff
@@ -2,14 +2,35 @@ package crowdstrike
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"os"
 
 	"github.com/crowdstrike/gofalcon/falcon"
 	"github.com/crowdstrike/gofalcon/falcon/client"
+	"golang.org/x/time/rate"
+
 	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
 )
 
+var ErrRateLimitExceeded error = errors.New("rate limit exceeded")
+var ErrResourceNotFound error = errors.New("resource not found")
+
+func getRateLimiter(ctx context.Context, d *plugin.QueryData) *rate.Limiter {
```

Comment:
Please remove this as per our earlier conversation, as it's not currently required